docs/src/pages/DesignPlaceholderPage/DesignPlaceholderPage.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import DocsPageBuilder from '../../components/DocsPageBuilder';
+import IntroBlurb from '../../components/IntroBlurb';
+import Paragraph from '../../components/Paragraph';
+
+const blurb = [
+  <IntroBlurb>Design guidelines needed!</IntroBlurb>,
```

Comment:
Could we maybe have something like?
```suggestion
  <IntroBlurb>Design guidelines do not currently exist for this component, but you can help!</IntroBlurb>,
```

Reply:
Or something more than `Design guidelines needed`

Reply:
Yeah I like it 😄